orchestra/migrations/0054_add_slack_ids.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-04-29 03:57
+from __future__ import unicode_literals
+
+from annoying.functions import get_object_or_None
+from django.db import migrations
+from django.conf import settings
+from slacker import Slacker
+
+import logging
+logger = logging.getLogger(__name__)
+
+def add_worker_slack_ids(apps, schema_editor):
+    Worker = apps.get_model('orchestra', 'Worker')
+    if not hasattr(settings, 'SLACK_EXPERTS_API_KEY'):
```

Comment:
check that the key is valid i.e. not an empty string or something, I think this is causing the migration to hang